### ISSUE TITLE
Please re-publish using gx after removing all files not tracked by git

### ISSUE THREAD

**Issue by Stebalien** [blang/semver#30]
I can't find a peer with current version pinned and,when I fetch the repo and pin it locally, I get a different hash.

**Comment by blang** [comment]
Published 3.5.1 2ee87856327ba09384cabd113bc6b5d174e9ec0f
Let me know if it works now, your problem might be that 4a1e882c79dcf4ec00d2e29fac74b9c8938d5052 was not published

**Comment by Stebalien** [comment]
3.5.1 works. FYI, when I tried to fetch 3.5.0, I separately:

1. Fetched the repo from github.
2. Checked out https://github.com/blang/semver/commit/fe96716bcf2ceafaf3ab0993900c5b53d538a9a0 (the gx publish commit for 3.5.0).
3. Ran `gx publish -f` to manually "seed" my own ipfs repo.

... and got a different hash.

Was https://github.com/blang/semver/commit/4a1e882c79dcf4ec00d2e29fac74b9c8938d5052 supposed to have been included in the gx release 3.5.0?

----

Regardless, it works now. Thanks!
